agent/sam.py

```
@@ -145,11 +145,26 @@ async def process_command(self, text: str, session_id: str = "default") -> str:
+            from langfuse_obs import emit_agency_observation
+            emit_agency_observation(
+                operation="sam_voice",
+                actor="user",
+                status="ok" if not response.startswith(self._fallback_response("")) else "fallback",
+                duration_ms=int((time.time() - _sam_start) * 1000),
+                input_text=text[:2000],
+                output_text=response[:2000],
+                metadata={"session_id": session_id, "turn": session.command_count},
+            )
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)